app/model/auth-message/auth-message_test.go

```diff
@@ -1,8 +1,9 @@
 package auth_message
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
nit: we should skip committing formatting changes IMO

Reply:
imports are rearranged by VS  Code (with default settings) on save

Reply:
Yeah I know that's the cause it's just adds a bit of overhead to the PR. I'm okay with merging them it's just something of note for future PR's if possible